test/cli/packages_cmd_test.go

```diff
@@ -294,7 +294,7 @@ func TestRegistryAuth(t *testing.T) {
 			args: args,
 			env: map[string]string{
 				"SYFT_REGISTRY_AUTH_AUTHORITY": host,
-				"SYFT_REGISTRY_AUTH_TOKEN":     "token",
+				"SYFT_REGISTRY_AUTH_TOKEN":     "my-token",
```

Comment:
Redactions apply to all output, so redacting `token` results in text being redacted containing this string.